### ISSUE TITLE
Ariba pubmlstget Error

### ISSUE THREAD

**Issue by Nilad** [sanger-pathogens/ariba#240]
Hi,

I report this error that i am encounter only on theses species ["Klebsiella pneumoniae", "Escherichia coli#2"]

```bash
ariba pubmlstget 'Klebsiella pneumoniae' /home/ME/kpneumoniae
WARNING: Found version 3.10.0 of spades which is too low! Please update to at least 3.11.0. Found it here: spades.py
Traceback (most recent call last):
  File "/usr/local/bin/ariba", line 298, in <module>
    args.func(args)
  File "/usr/local/lib/python3.5/dist-packages/ariba/tasks/pubmlstget.py", line 11, in run
    preparer.run()
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_ref_preparer.py", line 76, in run
    pubmlst.get_species_files(self.species, self.mlst_download_dir)
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_getter.py", line 121, in get_species_files
    self._download_profile_and_fastas(outdir, profile_url, fasta_urls)
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_getter.py", line 110, in _download_profile_and_fastas
    PubmlstGetter._rename_seqs_in_fasta(outfile + '.tmp', outfile)
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_getter.py", line 85, in _rename_seqs_in_fasta
    for seq in file_reader:
  File "/usr/local/lib/python3.5/dist-packages/pyfastaq/sequences.py", line 141, in file_reader
    raise Error('Error determining file type from file "' + fname + '". First line is:\n' + line.rstrip())
pyfastaq.sequences.Error: Error determining file type from file "/home/ME/kpneumoniae/pubmlst_download/rpoB.tfa.tmp". First line is:
<!DOCTYPE html>ariba pubmlstget 'Klebsiella pneumoniae' /home/ME/kpneumoniae
WARNING: Found version 3.10.0 of spades which is too low! Please update to at least 3.11.0. Found it here: spades.py
Traceback (most recent call last):
  File "/usr/local/bin/ariba", line 298, in <module>
    args.func(args)
  File "/usr/local/lib/python3.5/dist-packages/ariba/tasks/pubmlstget.py", line 11, in run
    preparer.run()
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_ref_preparer.py", line 76, in run
    pubmlst.get_species_files(self.species, self.mlst_download_dir)
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_getter.py", line 121, in get_species_files
    self._download_profile_and_fastas(outdir, profile_url, fasta_urls)
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_getter.py", line 110, in _download_profile_and_fastas
    PubmlstGetter._rename_seqs_in_fasta(outfile + '.tmp', outfile)
  File "/usr/local/lib/python3.5/dist-packages/ariba/pubmlst_getter.py", line 85, in _rename_seqs_in_fasta
    for seq in file_reader:
  File "/usr/local/lib/python3.5/dist-packages/pyfastaq/sequences.py", line 141, in file_reader
    raise Error('Error determining file type from file "' + fname + '". First line is:\n' + line.rstrip())
pyfastaq.sequences.Error: Error determining file type from file "/home/bacteriologie/Bureau/db_CNR/dbMLST/kpneumoniae/pubmlst_download/rpoB.tfa.tmp". First line is:
<!DOCTYPE html>
```

Ariba version and dependencies:

```
ariba version
WARNING: Found version 3.10.0 of spades which is too low! Please update to at least 3.11.0. Found it here: spades.py
ARIBA version: 2.12.0

External dependencies:
bowtie2	2.3.0	/usr/local/bowtie2-2.3.0/bowtie2
cdhit	4.7	/usr/local/cd-hit-v4.6.7/cd-hit
nucmer	3.1	/usr/bin/nucmer
spades	3.10.0	/usr/local/SPAdes-3.10.0/bin/spades.py

External dependencies OK: True

Python version:
3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609]

Python packages:
ariba	2.12.0	/usr/local/lib/python3.5/dist-packages/ariba/__init__.py
bs4	4.6.0	/usr/local/lib/python3.5/dist-packages/bs4/__init__.py
dendropy	4.3.0	/usr/local/lib/python3.5/dist-packages/dendropy/__init__.py
pyfastaq	3.17.0	/usr/local/lib/python3.5/dist-packages/pyfastaq/__init__.py
pymummer	0.10.3	/usr/local/lib/python3.5/dist-packages/pymummer/__init__.py
pysam	0.11.2.2	/usr/local/lib/python3.5/dist-packages/pysam/__init__.py

Python packages OK: True

Everything looks OK: True
```

I hope that can be usefull for you to resolve this problem.

**Comment by kpepper** [comment]
Hi **@Nilad**, I would think that because Ariba has found Spades installed on your system, it checks the version. As an easy fix, just upgrade the Spades version to 3.11.0 or above and it should be happy.

**Comment by Nilad** [comment]
@kpepper fixed. thank for your reply.
